tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/types/UniqueGiftBackdropColors.kt

```diff
@@ -0,0 +1,24 @@
+package dev.inmo.tgbotapi.types
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * This object describes the colors of the backdrop of a unique gift.
+ *
+ * @param centerColor The color in the center of the backdrop in RGB format
+ * @param edgeColor The color on the edges of the backdrop in RGB format
+ * @param symbolColor The color to be applied to the symbol in RGB format
+ * @param textColor The color for the text on the backdrop in RGB format
+ */
+@Serializable
+data class UniqueGiftBackdropColors(
+    @SerialName(centerColorField)
+    val centerColor: Int,
```

Comment:
It is better for all colors to use `RGBColor` abstraction from `utils` of `tgbotapi` where it is applicable